nikola/post.py

```diff
@@ -1089,6 +1090,15 @@ def source_ext(self, prefix=False):
         else:
             return ext
 
+    def should_hide_title(self):
+        """Return True if this post's title should be hidden. Use in templates to manage posts without titles."""
+        return self.title().strip() in ('NO TITLE', '') or self.meta('hidetitle') or \
```

Comment:
Is there any specific reason why `NO TITLE` is used? Is a blank string not enough?

Reply:
If you import a blog from WordPress, and if there are posts without titles in the original, then they are given the "NO TITLE" string in Nikola.

Reply:
Yeah, decisions made 10 years ago :-D